pipeline_dp/__init__.py

```diff
@@ -14,4 +14,3 @@
 from pipeline_dp.pipeline_backend import SparkRDDBackend
 
 __version__ = '0.0.1rc0'
-
```

Comment:
Is this intended?

Reply:
Yeah, this is because of "make format" and for some reason that fixed linting on github workflow, which is great.